### PR TITLE
Fixes #8044: Some messages in the Settings page end with a comma

### PR DESCRIPTION
**Pull request by RaphaelGauthier**
https://www.rudder-project.org/redmine/issues/8044

### CODE REVIEW

File: rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala [hunk 1]
```diff
@@ -112,7 +112,10 @@ class PropertiesManagement extends DispatchSnippet with Loggable {
 
     // Check that there is some modification to enabled/disable save
     def check() = {
-      S.notice("updateChangeMsg","")
+      noModif match{
```
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala]: On a boolean, you can do a if (noModif) {} else {} 

Ok i said it was better in scala to do pattern matching but not on everything ;)

Comment (RaphaelGauthier) [inline rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala]: This was a pattern matching training haha... I will correct this ;)

**Comment by RaphaelGauthier** [comment]
PR updated

File: rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala [hunk 2]
```diff
@@ -257,7 +259,9 @@ class PropertiesManagement extends DispatchSnippet with Loggable {
                   )
 
     def check() = {
-      S.notice("updateWorkflow","")
+      if(!noModif){
+        S.notice("updateChangeMsg","")
```
Comment (VinceMacBuche) [inline rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala]: It should be updateWorkflow here no ? and on all other ids below ?

**Comment by RaphaelGauthier** [comment]
PR updated

**Pull request merged**